deploy/Makefile

```diff
@@ -55,10 +55,10 @@ postgres_logs:
 postgres_psql:
 	@DOCKER_HOST=$(DOCKER_HOST) docker compose --project-name ${PROJECT_NAME} --env-file $(ENV_FILE) --file docker-compose.yml --file docker-compose.override.yml --profile postgres exec postgres psql -h localhost -U ${POSTGRES_USER} postgres
 
-create_db_setup_script:
+create_db_setup_script:  # NOTE: This script assumes we have an existing instance to setup the DB, maybe we can rather just let this run locally?
```

Comment:
This particular script is quite simple and doesn't need any deps, would it just not be easier to call it directly instead of docker?

OR

We can just have both, with a note in the README. Having to first configure the env to get the DB setup to work and then configuring env again once the DB setup is done can be very confusing.

Ideally DB setup & Druid setup should be completely isolated from web setup imo.

Reply:
The script assumes you have python set up correctly - so much easier to avoid that nightmare and run it in Docker i.m.h.o.

The flow is super confusing, I agree, snake eating it's own tail.

Do we have bandwidth to correct it? Or just sticking to hellu confusing instructions.

Reply:
Nah, confusing instructions it is for now.